app/index.html

```diff
@@ -8,34 +8,42 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <header>
-        <div class="center align-left" id="logo">
-            <a href="#" id="logo-text" class="primary-text-color">Welcome</a>
-        </div>
-        <nav class="center align-right">
-            <img id="menu" src="assets/menu_button.png" alt="">
-        </nav>
-    </header>
-    <main id="content">
-        <section>
-            <h1 class="center-text primary-text-color primary-font heading u-margin">Hey there. I&apos;m Mariane <br><span class="secondary-text-color">I&apos;m a software developer</span> </h1>
-
-            <p class="paragraph primary-text-color"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don&apos;t hestiate to contact me.</p>
-        </section>
-        
-        <section>
-            <ul class="social-icons">
-                <li><a href="#"><img src="./assets/social-icons/twitter.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/Linkedin_icon.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/medium.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/github.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/angellist.png" alt=""></a></li>
-            </ul>
-        </section>
-
-        <a href="" class="scroll-down-btn"></a>
-           
-    </main>
+    <div id="container">
```

Comment:
- [ ] Your whole project has a horizontal overflow.  Kindy please correct it, so the page stretches correctly on multiple screens.

Reply:
[Resolved] horizontal overflow issue

Reply:
Addressed 🤓